Upload uperf results

aakarshg opened the pull request.

dustinblack committed:
Restructure deploy dir and default to ripsaw namespace

- Restructured deploy dir
- All resources created will be bound to ripsaw namespace
- Updated test scripts to look at ripsaw namespace
- Moved namespace file into resources dir.
- Added group_vars to dockerfile
- Fixed ycsb test deleting couchbase cr

Co-authored-by: aakarshg <aakarsh.g2012@gmail.com>

jtaleric committed:
Migrating to benchmark

Moving our resource to benchmark from bench.

dustinblack committed:
Removing uneeded rbac

Turns out there was a bunch of rbac stuff that was leftover from before moving to operatorhub.io, so those tasks and components have been removed.

aakarshg committed:
Updated operator image

The operator image has been updated to use cloudbulldozer/ripsaw
from quay.

brianr27 committed:
updating docs with name change

jtaleric committed:
Remove images from personal repos

Move workload images to shared namespace

aakarshg committed:
Upload uperf results

- Added new role result which will upload base64 encoded data to es
- Currently only uperf supports it

Co-authored-by: Joe (rook) <joe.talerico@gmail.com>

jtaleric commented on roles/uperf-bench/tasks/main.yml:
```diff
@@ -94,20 +94,25 @@
                   - "cat /tmp/uperf-test/uperfXML;
                      export h={{item.status.podIP}};
-                     uperf -v -x -a -m /tmp/uperf-test/uperfXML -i 1"
+                     uperf -v -x -a -m /tmp/uperf-test/uperfXML -i 1 >> {{results.path}}/{{ '%d%m%Y_%H%M%S' | strftime }}-uperf"
```
Use tee.

jtaleric commented on roles/uperf-bench/tasks/main.yml:
```diff
@@ -47,8 +47,8 @@
             app : uperf-bench-server
         spec:
```
Move these to templates. That way we can put the logic there. Use sysbench as an example.

jtaleric committed:
Bring your own workload (#66)

* Bring your own workload

This work will allow users to execute their own workloads against their
own containers they provide.
Updated links in docs

jtaleric committed:
Better structure to our CR

This commit will provide a structure to our CR.

Closes #7

aakarshg committed:
Fixed ycsb test script

No more race condition to wait for a pod to enter Ready state.
